README.md

```diff
@@ -2,6 +2,9 @@
 
 Module provides an extension plugin for Spryker's Monitoring module.
 
+[![Build Status](https://travis-ci.org/spryker-eco/tideways.svg?branch=master)](https://travis-ci.org/spryker-eco/tideways)
```

Comment:
wrong order
should be after Header  directly

Reply:
Fixed.